pillar/fluentd/tls_forward.jinja

```diff
@@ -0,0 +1,23 @@
+{% set tls_forward = {
+    'directive': 'match',
+    'directive_arg': '**',
+    'attrs': [
+        {'@type': 'forward'},
+        {'transport': 'tls'},
+        {'tls_client_cert_path': salt.sdb.yaml.get('fluentd:cert_path')},
+        {'tls_client_private_key_path': salt.sdb.yaml.get('fluentd:cert_key_path')},
```

Comment:
These calls wouldn't be `sdb.yaml.get` it would be `sdb.get('sdb://yaml/fluentd:cert_key_path')`, etc.